### ISSUE TITLE
Idea: Shortcut for QR-Code scanning 

### ISSUE THREAD

**Issue by Echolon** [kriztan/Pix-Art-Messenger#190]
_Hi, I just copy pasted this from the same issue I created on Conversations profile. Just in case this is going to be closed without application. https://github.com/siacs/Conversations/issues/2896_

-------------------------------------------------
**Version:** 1.23.8 and 2.0.0
**Device:** Samsung S3 and OnePlus 2
**Android Version:** LineageOS
**Conversations source:** PlayStore, F-Droid

#### Issue

It's pretty hard and obstacle-like way to find the QR-Code via the Share-Button. Most people probably guess to only share their ID from there. Today me and a friend (and we are both engineers ;) ) couldn't find the QR-Code for verification (before asking in the group chat). 
To kind of hide it there makes it even more hard for most people to "easy & voluntary" verify their fingerprints. We both discussed my idea to improve here, thus I like to present it to you.

#### Idea

I wonder this did came up earlier. Anyway, we would like to suggest to just put a shortcut on each fingerprint field, as the screen-shots below show:

#### _Previous layout_ 
(Sry it's a screen still from 1.23.8)
![qr1](https://user-images.githubusercontent.com/29168011/37867649-e8a27540-2f9b-11e8-8678-3abbcfd03e29.jpg)

#### _Our idea:_
![qr2](https://user-images.githubusercontent.com/29168011/37867652-ebd2a2d0-2f9b-11e8-98ef-244ab900803d.jpg)

If the copy-button is in the way, it probably should be dropped or hidden in the previous method by long-pressing the field. I'm questioning: How often do you copy this in comparison to the number of 0 verification your intend to do?

Hope you agree that this is a more intuitive method. A small "Verify"-text below this button may provide additional help understanding.

Kind regards, feel free to vote!

**Comment by kriztan** [comment]
This is a nice idea. I'll try to integrate it for upcomming version 2.0

------ Originalnachricht ------
Von: "Echolon" <notifications@github.com>
An: "kriztan/Pix-Art-Messenger" <Pix-Art-Messenger@noreply.github.com>
Cc: "Subscribed" <subscribed@noreply.github.com>
Gesendet: 24.03.2018 20:37:12
Betreff: [kriztan/Pix-Art-Messenger] Idea: (#190)

**Comment by Echolon** [comment]
@kriztan Hi, happy to hear you like it - I suggest we wait until this will be implemented (or rejected ;) ) in Conversations - then you can decided (otherwise will be double work)

**Comment by Echolon** [comment]
_Made the same point on Conversations Github issue. See link above!_

-------------------------------------------------

Hi,

me again. I just saw that Conversations already has a solution for this - maybe came with C2.0 once I updated. Unfortunately, this is only usable for fingerprints I like to scan, not for those to show from my side. I'm talking about this from a contact's profile:

![screenshot_20180327-175556](https://user-images.githubusercontent.com/29168011/37983938-2a6ed4e0-31f5-11e8-9788-ddd8aa8be8ef.png)

Would be great to see this my own fingerprints, too. Still, I think the QR-Code-Symbol is more intuitive, but okay.

Cheers
